utils/viewport.js

```diff
@@ -0,0 +1,3 @@
+export function isExtraSmall() {
+	return window && window.innerWidth < 768;
```

Comment:
We have a similar check in `editor/state/store-defaults.js` we should be consistent, I think.